src/compass/utils/runconfig.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from itertools import cycle
+import os
+from types import SimpleNamespace
+
+import journal
+import yamale
+from ruamel.yaml import YAML
+
+from compass.utils import helpers
+from compass.utils.wrap_namespace import wrap_namespace
```

Comment:
It seems that this function is not contained in the `utils` folder. Make sure to bundle it with this PR for completeness